csug/syntax.stex

```diff
@@ -327,6 +327,32 @@ For example:
 \scheme{datum->syntax-object} is identical to the Revised$^6$ Report
 \scheme{datum->syntax}.
 
+%----------------------------------------------------------------------------
+\entryheader
+\formdef{quote-syntax}{\categorysyntax}{(quote-syntax \var{datum})}
+\returns a syntax object
+\listlibraries
+\endentryheader
+
+A \scheme{quote-syntax} expression is like a \scheme{quote}
+expression except that contextual information associated with
+the datum is retained.
+
+Unlike \scheme{syntax}, \scheme{quote-syntax} does not substitute
+pattern variables bound by \scheme{with-syntax} or
+\scheme{syntax-case}.
+
+(This form is identical to the Revised$^4$ Report
+\scheme{syntax}.)
```

Comment:
I'd leave out the R4RS reference, just because R4RS has generally be referenced in the manual. (I don't know for certain that there are other places where a similar comment would apply, but it seems likely.)